app/brand.go

```diff
@@ -77,3 +77,19 @@ func (a *App) GetBrandImage() ([]byte, *model.AppError) {
 
 	return img, nil
 }
+
+func (a *App) DeleteBrandImage() *model.AppError {
+	filePath := BRAND_FILE_PATH + BRAND_FILE_NAME
+
+	fileExists, err := a.FileExists(filePath)
+
+	if err != nil {
+		return err
```

Comment:
Is err here an AppError?

Reply:
Yes it is. 

Otherwise It wouldn't compile, no ?

https://github.com/mattermost/mattermost-server/blob/0cc61b4faf6f4270f834e2e52a848aed3a8b7316/app/file.go#L84